test/ask-permission.js

```diff
@@ -0,0 +1,58 @@
+import {spawn} from 'child_process';
+import test from 'ava';
+
+test.cb('skip in TTY mode', t => {
+	t.plan(1);
+
+	const insProcess = spawn('node', [
```

Comment:
Just to be sure, if I do this the `execa` will be rejected. It is OK to wrap it with a `try catch` right?
```
test('skip in TTY mode', async t => {
	try {
		await execa('node', ['./test/fixtures/sub-process.js']);
	} catch (err) {
		t.is(err.code, 145);
	}
});
```